### PR TITLE
test: Quiet down pip (errors/warnings will still show)

### PR DESCRIPTION
**Pull request by alex-hofsteede**
> The rule of silence, also referred to as the silence is golden rule, is an important part of the Unix philosophy that states that when a program has nothing surprising, interesting or useful to say, it should say nothing. It means that well-behaved programs should treat their users' attention and concentration as being valuable and thus perform their tasks as unobtrusively as possible. That is, silence in itself is a virtue.

TLDR, scrolling through thousands of lines of `pip install` boilerplate in travis test runs is annoying

### CODE REVIEW

**Review by mattrobenolt — APPROVED** [review]

**Comment by mattrobenolt** [comment]
An alternative is we could just set an environment variable in all of Travis. This way it doesn’t change behavior outside of that, and we don’t need to remember every single invocation.

I think something like `PIP_QUIET=1` works. I’d need to check. But all command line flags have env car equivalents.

**Comment by mattrobenolt** [comment]
Yeah, just confirmed, `PIP_QUIET=1` could work instead of slamming down `-q` everywhere, then we can just set this in the `.travis.yml` file.

**Comment by alex-hofsteede** [comment]
Good point, made that change. 

The build logs on this PR already look so much better!

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ ifneq "$(wildcard /usr/local/opt/openssl/lib)" ""
 	LDFLAGS += -L/usr/local/opt/openssl/lib
 endif
 
-PIP = LDFLAGS="$(LDFLAGS)" pip
+PIP = LDFLAGS="$(LDFLAGS)" pip -q
```
Comment (mattrobenolt) [inline Makefile]: I'd remove this then too, and purely rely on `PIP_QUIET` that you're setting as the env var. I'd argue, if someone wnats this locally or whatever, they can also set the env var.

**Pull request merged**